caffe2/CMakeLists.txt

```diff
@@ -671,12 +657,12 @@ endif()
 add_library(torch_cpu ${Caffe2_CPU_SRCS})
 torch_compile_options(torch_cpu)  # see cmake/public/utils.cmake
 
-if (LLVM_FOUND)
+if (USE_LLVM AND LLVM_FOUND)
```

Comment:
can we just assert that `USE_LLVM` is true 
`USE_LLVM` should always be true if `LLVM_FOUND` is true since we don't even search for llvm unless `USE_LLVM` is set to true?

Reply:
Here I'm just following how OPENMP is checked and enabled.